numpy/core/tests/test_scalarinherit.py

```diff
@@ -68,7 +68,7 @@ class Mystr(str, np.generic):
     def test_char_repeat(self):
         np_s = np.string_('abc')
         np_u = np.unicode_('abc')
-        np_i = np.int(5)
+        np_i = np.int_(5)
         res_s = b'abc' * 5
         res_u = u'abc' * 5
         assert_(np_s * np_i == res_s)
```

Comment:
This test now fails, because `np.string_ * np.int_` isn't legal in python 3. It seems the test was supposed to be testing this all along, but was written badly.